kvmagent/kvmagent/plugins/ha_plugin.py

```
@@ -819,11 +1047,15 @@ def get_current_completion(_, content):
             finally:
                 read_complete.set()
 
-        length = self.ioctx.stat(self.heartbeat_object_name)[0]
+        length, used_time, err = self.get_heartbeat_object_length()
+        if err is not None:
+            logger.debug("failed to get ceph object %s length, %s" % (self.heartbeat_object_name, err))
+            return None, None
+
         completion = self.ioctx.aio_read(self.heartbeat_object_name, int(length), 0, get_current_completion)
```

Ruff (0.12.2)

Local variable `completion` is assigned to but never used

Remove assignment to unused variable `completion`

(F841)